trieste/utils/pareto.py

```diff
@@ -73,16 +74,22 @@ class Pareto:
     The latter is needed for certain multiobjective acquisition functions.
     """
 
-    def __init__(self, observations: TensorType):
+    def __init__(self, observations: TensorType, generic_strategy: bool = False):
```

Comment:
what's the use case for `generic_strategy`? Why would a user want to use the generic strategy when the 2D strategy works (and is presumably quicker?)?

Reply:
just add a choice to user to compares those two. but it seems not necessary to add this. I'll remove it

Reply:
Just thinking about it again. it is needed to test if generic_strategy also valid for 2d case. if we remove it we should also remove 2d test case for divide_conquer_nd. what do you think?

Reply:
I'd say remove it. I believe it's not a good idea to expose functionality for the sake of testing - you'll end up having to maintain a larger API.

That said, I'm not sure about `divide_conquer_nd` being public at all. It's already called at initialisation and the result exposed via `.bounds`

Reply:
okay

Reply:
removed it